rskj-core/src/main/java/org/ethereum/rpc/dto/TransactionReceiptDTO.java

```diff
@@ -71,6 +72,7 @@ public TransactionReceiptDTO(Block block, TransactionInfo txInfo, SignatureCache
         transactionHash = receipt.getTransaction().getHash().toJsonString();
         transactionIndex = HexUtils.toQuantityJsonHex(txInfo.getIndex());
         logsBloom = HexUtils.toUnformattedJsonHex(txInfo.getReceipt().getBloomFilter().getData());
+        type = "0x0";
```

Comment:
This value doesn't change so it is better to use a static constant instead of using a var for each object.

Reply:
Yep, makes total sense. thanks